snapcraft/tests/test_yaml.py

```diff
@@ -317,7 +300,6 @@ def test_config_expands_filesets(self, mock_loadPlugin):
 version: "1"
 summary: test
 description: test
-icon: my-icon.png
```

Comment:
They could be. These small refactors were to simply make the tests consistent with the ones I was adding rather than being consistent the other way and adding more cruft.